### ISSUE TITLE
keyboard focus is not visible on the settings dropdown and checkboxes present in the setting dropdown. 

### ISSUE THREAD

**Issue by msftedad** [TypeStrong/typedoc#2556]
## Expected Behavior

keyboard focus should be visible on the 'settings' dropdown and on the checkboxes present in the setting dropdown.

## Actual Behavior

While navigate to settings dropdown, keyboard focus is not visible on the 'settings' dropdown and on the checkboxes present in the setting dropdown.

https://github.com/TypeStrong/typedoc/assets/93735775/4e988358-a364-4190-9dc4-871fcbf1ddad

## Steps to reproduce the bug

1. Open [TypeDoc - v0.25.13](https://typedoc.org/api/) in edge browser.
2. Navigate to settings dropdown and observe the keyboard on it, and after invoking it observe the keyboard focus on the checkboxes.

## Environment

-   Typedoc version:  0.25.13
-   TypeScript version:
-   Node.js version:
-   OS: 23H2 Build 22631.3527

**Comment by Gerrit0** [comment]
Fixed with TypeDoc 0.26, which is releasing 2024/06/21
